aptos/sources/Implementation.move

```diff
@@ -0,0 +1,19 @@
+module Wormhole::Implementation{
+    use 0x1::event::{Self};
+    use 0x1::signer::{address_of};
+    //use Wormhole::State::{WormholeMessage};
+    use Wormhole::State::{nextSequence, setNextSequence, WormholeMessageHandle, publishMessage};
+    
+    
+    
+    // TODO: how to add fee? 
+    //require(msg.value == messageFee(), "invalid fee");
+
+    //sequence = useSequence(msg.sender);
+    // emit log
+    //emit LogMessagePublished(msg.sender, sequence, nonce, payload, consistencyLevel);
```

Comment:
Consider moving this PR to a draft if it isn't quite ready for review yet.

Please read [these docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft) on how to do that.